vetiver/handlers/statsmodels.py

```diff
@@ -0,0 +1,67 @@
+import pandas as pd
+
+from ..meta import _model_meta
+from .base import BaseHandler
+
+sm_exists = True
+try:
+    import statsmodels
+except ImportError:
+    sm_exists = False
+
+
+class StatsmodelsHandler(BaseHandler):
+    """Handler class for creating VetiverModels with statsmodels.
+
+    Parameters
+    ----------
+    model : statsmodels
+        a trained and fit statsmodels model
+    """
+
+    model_class = staticmethod(lambda: statsmodels.base.wrapper.ResultsWrapper)
```

Comment:
for models with a `predict` method, using this class seems to be sufficient to identify `statsmodels` models